src/Tizen.Uix.VoiceControlManager/Tizen.Uix.VoiceControlManager/VoiceCommand.cs

```diff
@@ -0,0 +1,220 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using static Interop.VoiceControlManager;
+using static Interop.VoiceControlCommand;
+
+namespace Tizen.Uix.VoiceControlManager
+{
+    /// <summary>
+    /// Enumeration for the command format.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public enum CommandFormat
+    {
+        /// <summary>
+        /// The fixed command format.
+        /// </summary>
+        Fixed = 0,
+        /// <summary>
+        /// The fixed and variable fixed command format.
+        /// </summary>
+        FixedAndVFixed = 1,
+        /// <summary>
+        /// The variable fixed and fixed command format.
+        /// </summary>
+        VFixedAndFixed = 2,
+        /// <summary>
+        /// The fixed and non-fixed command format.
+        /// </summary>
+        FixedAndNonFixed = 3,
+        /// <summary>
+        /// The non-fixed and fixed command format.
+        /// </summary>
+        NonFixedAndFixed = 4,
+        /// <summary>
+        /// The action command
+        /// </summary>
+        Action = 5,
+        /// <summary>
+        /// The partial matched command
+        /// </summary>
+        Partial = 6,
+        /// <summary>
+        /// Undefined
+        /// </summary>
+        Undefined = -1
+    };
+
+    /// <summary>
+    /// This class represents a voice command.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class VoiceCommand
```

Comment:
You need to implement IDisposable interface because it has SafeCommandHandle

Reply:
Fixed.